Automattic/jetpack
# DonutMeter: Expose DonutMeter to clients of the component library.

#### Changes proposed in this Pull Request:


Renames the containing folder to match the final component name and exposes the DonutMeter to clients of the component library.

#### Other information:


#### Does this pull request change what data or activity we track or use?



No

#### Testing instructions:






1. Run `cd projects/js-packages/storybook && npm run storybook:dev` and confirm that the DonutMeter component is still visible in Storybook. It should be under **JS Packages → Components → DonutMeter**.
2. Run `jetpack test` against `js-packages/components` and confirm tests are still passing.
